client/editor/map_editor.cpp

```diff
@@ -27,14 +29,28 @@ map_editor::map_editor(QWidget *parent)
   setAutoFillBackground(true);
   setVisible(false);
 
-  ui.label_status->setText(_("MAP EDITOR<br/>This is WIP!"));
+  QIcon::setThemeName(QStringLiteral("icons"));
```

Comment:
This is a global thing, it should probably be done in the client boot code.

Reply:
It's set elsewhere in client_main() I think. I'll remove.